src/electionguard-ui/ElectionGuard.UI/ViewModels/BallotUploadViewModel.cs

```diff
@@ -58,8 +58,8 @@ public partial class BallotUploadViewModel : BaseViewModel
         {
             var record = await _manifestService.GetByElectionIdAsync(value);
             using var manifest = new Manifest(record?.ManifestData);
-            _manifestHash = new(manifest.CryptoHash());
             _internalManifest = new InternalManifest(manifest);
+            _manifestHash = new(_internalManifest.ManifestHash);

```

Comment:
Changed to get the hash from the internal manifest instead of from the manifest.